src/frontend/komponenter/Fagsak/Behandle/FastsettVedtak.tsx

```diff
@@ -1,143 +1,100 @@
-                                                oppdatertBarnBeregning: {
-                                                    ...barnBeregning.verdi,
-                                                    beløp: parseInt(event.target.value, 10),
-                                                },
-                                            },
-                                            type: 'SETT_BARNAS_BEREGNING',
-                                        });
-                                    }}
-                                />
-
-                                <Input
-                                    bredde={'L'}
-                                    label={'Startdato'}
-                                    value={barnBeregning.verdi.startDato}
-                                    placeholder={'DD.MM.YY'}
-                                    onChange={event => {
-                                        dispatch({
-                                            payload: {
-                                                index,
-                                                oppdatertBarnBeregning: {
-                                                    ...barnBeregning.verdi,
-                                                    startDato: event.target.value,
-                                                },
-                                            },
-                                            type: 'SETT_BARNAS_BEREGNING',
-                                        });
-                                    }}
-                                />
-                            </SkjemaGruppe>
-                        );
-                    }
-                )}
-                <Normaltekst
-                    children={`Totalsum: ${state.barnasBeregning.reduce(
-                        (sum: number, barnBeregning: IFelt<IBarnBeregning>) =>
-                            sum + barnBeregning.verdi.beløp,
-                        0
-                    )} kr`}
-                />
-            </Panel>
-
-            <Undertittel children={'Hjemler'} />
-            <Normaltekst children={'Vedtaket er fattet etter § 2 og § 11 i barnetrygdloven.'} />
-
-            <Undertittel children={'Resultat'} />
-            <RadioPanelGruppe
-                name="behandlingsresultat"
-                legend="Behandlingsresultat"
-                radios={[
-                    { label: 'Innvilget', value: 'innvilget', id: 'innvilget' },
-                    { label: 'Avslått', value: 'avslått', id: 'avslått' },
-                ]}
-                checked={state.behandlingsresultat}
-                onChange={(event: any) => {
-                    dispatch({ type: 'SETT_BEHANDLINGSRESULTAT', payload: event.target.value });
-                }}
+            <FastsettVedtakSkjema
+                opprettelseFeilmelding={opprettelseFeilmelding}
+                visFeilmeldinger={visFeilmeldinger}
             />
+
             <Nesteknapp
```

Comment:
Shouldn't we hide the next button, or show another button when there is an error?

Reply:
The design system are pretty much against disabling og hiding buttons, so we show them and give the user the most intuitive message whenever something is wrong.